File: 1_class_object.cpp
#include <bits/stdc++.h>
using namespace std;

class Rectangle
{
private:
    float length;
    float width;

public:
    Rectangle(float l, float w) : length(l), width(w)
    {
        cout << "Rectangle object created !! length : " << length << "; width: " << width << endl;
    }
    float area()
    {
        return length * width;
    }
    float perimeter()
    {
        return 2 * (length + width);
    }

    ~Rectangle()
    {
        cout << "Rectangle object destroyed" << endl;
    }
};

int main()
{
    Rectangle rectangle(5.0, 3.0);
    cout << "Area is: " << rectangle.area() << endl;
    cout << "Perimeter is: " << rectangle.perimeter() << endl;

    return 0;
}

File: 1_class_object.py
class Rectangle:
    def __init__(self, length, width):
        self.length = length
        self.width = width

    def area(self):
        return self.length * self.width

    def __str__(self):
        return f"Rectangle object created !! length: {self.length}; width: {self.width}"

    def perimeter(self):
        return 2 * (self.length + self.width)

    def __del__(self):
        print("Rectangle object destroyed")

def main():
    rectangle = Rectangle(5.0, 3.0)
    print("Area is:", rectangle.area())
    print("Perimeter is:", rectangle.perimeter())

if __name__ == "__main__":
    main()
